lib/Mojo/WebSocketProxy.pm

```diff
@@ -0,0 +1,252 @@
+package Mojo::WebSocketProxy;
+
+use strict;
+
+our $VERSION = '0.01';
+
+1;
+
+__END__
+
+=head1 NAME
+
+Mojo::WebSocketProxy::Dispatcher
+
+=head1 SYNOPSYS
+    
+    # lib/your-application.pm
+ 
+    use base 'Mojolicious';
+ 
+    sub startup {
+        my $self = shift;
+        $self->plugin(
+            'web_socket_proxy' => {
+                actions => [
+                    ['json_key', {some_param => 'some_value'}]
+                ],
+                base_path => '/api',
+                url => 'http://rpc-host.com:8080/',
+            }
+        );
+   }
+
+Or to manually call RPC server:
+    
+    # lib/your-application.pm
+ 
+    use base 'Mojolicious';
+ 
+    sub startup {
+        my $self = shift;
+        $self->plugin(
+            'web_socket_proxy' => {
+                actions => [
+                    [
+                        'json_key', 
+                        {
+                            instead_of_forward => sub {
+                                shift->call_rpc({  
+                                    args => $args,
+                                    method => $rpc_method, # it'll call 'http://rpc-host.com:8080/rpc_method'
+                                    rpc_response_cb => sub {...}
```

Comment:
I think it's better to use long name to avoid get confused, because there could be many hooks and callback.